### ISSUE TITLE
Find a bug / suggest a feature

### ISSUE THREAD

**Issue by championswimmer** [coding-blocks/coding-blocks.github.io#29]
Win 20 bounty points by doing either - 
 - **Find a bug**

For this you have to mention expected result, and actual outcome, and post screenshots (if required) to substantiate. Vague bug reports that can't be reproduced will not be entertained

or 

 - **Suggest a feature**

Suggest a feature that can be added. Flesh it out with technical details, and steps required to implement. Post mocks (if required) to explain the feature. It will be considered for bounty if the maintainers consider the feature suggestion valid and decide to work on it. 

----

For either case, file a bug or suggest a feature by opening a new issue.

**Comment by hereisnaman** [comment]
Replace the current testimonial slider with a masonry grid, with a lightbox, containg testimonial data from [THESE](https://www.facebook.com/pg/codingblocksindia/photos/?tab=album&album_id=1762909133927422) testimonial images. This way we can add many more reviews to the pages, all visible at the same time; but the same is not possible with the current slider. 

More Reviews=>More schema data=>Google loves you.

**Comment by championswimmer** [comment]
On 24 April 2017 at 14:24, Naman Kumar <notifications@github.com> wrote:

> masonry grid



SHould we do it in a separate reviews page ?

**Comment by hereisnaman** [comment]
Yeah we can even design a separate testimonial page and arrange reviews category wise, then in that case masonry won't be a nice suggestion, we can have any other design for it.

On the review page we can use microdata and put all the required information on the page. And then  we can put just the aggregate rating schemas on course pages rather than puting reviews schemas, like the below ss and link to the testimonial page:

![](http://i63.tinypic.com/2emhg1f.jpg)

**Comment by hereisnaman** [comment]
@championswimmer looking forward to work the testimonial page, please create a separate issue. 

Also bug found at Interview Prep Page and resolved here: https://github.com/coding-blocks/coding-blocks.github.io/pull/61

**Comment by prateek27** [comment]
@hereisnaman  Yes we should have student testimonial page, along with a filter to sort testimonials course wise.

**Comment by hereisnaman** [comment]
@prateek27 sure sir, in pipeline.

**Comment by JeetuGoel** [comment]
We can develop a feature, so that the video of the previous class is played online, a few hours before the next class, and the students can login to watch it, which makes it easy for everyone to revise the lectures and also for those who missed the classes for some reason.

**Comment by JeetuGoel** [comment]
![image](https://user-images.githubusercontent.com/25008517/27514246-27b5af20-59a2-11e7-9e13-cb0dbcd6025c.png)

Contents of navbar are not aligned properly.

**Comment by ArushiSinghal** [comment]
@championswimmer @prateek27 
when on the page https://codingblocks.com/resources/ and clicked on the Coding block icon it is not redirecting to the home page.
![imageedit_1_8900677811](https://user-images.githubusercontent.com/20858804/27518990-441f873c-5a09-11e7-91b6-3b3c73da7433.gif)

**Comment by ArushiSinghal** [comment]
For good design in the competition tag if we select any option (hackerblock or BOSS) it should open in new tab.
![imageedit_4_4362710047](https://user-images.githubusercontent.com/20858804/27519393-dcfd31e6-5a10-11e7-811d-bc21edf3a2a8.gif)

**Comment by AshishkrGoyal** [comment]
@championswimmer we can add a AJAX preloader for our site, on slow network it will enhance user experience.

**Comment by till-tomorrow** [comment]
![cb resources menu](https://user-images.githubusercontent.com/21954525/40033073-63bff982-5814-11e8-80c3-da667430602c.JPG)

The contents of the menu are hardly visible on this page.
change the font color to red(or something you want to).

**Comment by Yagya-Chaudhary** [comment]
![screenshot from 2018-05-18 21-29-03](https://user-images.githubusercontent.com/23234409/40245282-d6705924-5ae2-11e8-9c6d-4957900c42f3.png)
Titles like Resources and Campus Ambassadors are overlapping

**Comment by Rupeshiya** [comment]
Add feature for feeedback .
Refference to #536

**Comment by tinkrboy** [comment]
when on the page https://codingblocks.com/resources/ and clicked on the readmore button of every section then it is not using correct route path.
For instance: i am on javascript section of resources and click on Read More button then it is redirecting to https://codingblocks.com/resources/resources/javascript/.
**Expected Result:**  It should open https://codingblocks.com/resources/javascript/ 
**Actual Outcome:**  GitHub page not found

**Comment by crusader2000** [comment]
We can add hero sliders for the online and classroom courses for the main site. We can use third party scripts for this.

**Comment by P-riyanka-prasad** [comment]
when clicking on the **link** 
the text get overwritten resources and campus ambassadors 
[link](https://codingblocks.com/resources)

![Annotation 2020-05-16 083009](https://user-images.githubusercontent.com/59612128/82109423-51486480-9753-11ea-828f-cc22c0c95c92.png)

**Comment by chiraggoel360** [comment]
![Screenshot_2020-05-21-13-21-16-36](https://user-images.githubusercontent.com/51852665/82536395-05882780-9b66-11ea-92d2-90ec27101b25.jpg)
Need to login again and again

**Comment by chiraggoel360** [comment]
![Screenshot_2020-05-21-13-24-32-10](https://user-images.githubusercontent.com/51852665/82536698-7d565200-9b66-11ea-99dd-cbf0ccf7c3ec.jpg)
Cant load the google map even the location is enabled

**Comment by chiraggoel360** [comment]
![Uploading 15900491207388945696221576018051.jpg…]()
There must be display of link

**Comment by chiraggoel360** [comment]
![15900491680058082781603548206758](https://user-images.githubusercontent.com/51852665/82538792-e8555800-9b69-11ea-9aca-8df7b3887646.jpg)
There must be link to copy

**Comment by ADITIJ700** [comment]
@championswimmer 

1. **SUGGESTIONS**

1) ### **CHAT BOT CLOSE SIGN TO BE KEPT ON RIGHT**
![chatbox](https://user-images.githubusercontent.com/36340264/83038882-18fe2b80-a05b-11ea-8774-1847638ba0f4.png)

- It's a genral convention to close a window on the right side,hence it terms of UX ,the close sign when kept on the right,would be more beneficial.

 2)  ### **break between the content**
[https://bytecast.codingblocks.com/](url)

- the two headers and their date would look better,if there is a break/space  given.

2. **BUGS**

1)  ### ** HOME PAGE FIRST DIVISION CONTAINS TWO PARAGRAPHS CONFLICTING WITH EACH OTHER**
![home_page](https://user-images.githubusercontent.com/36340264/83039582-fcaebe80-a05b-11ea-9c06-259f30699e0a.png)

- 
![home(1)](https://user-images.githubusercontent.com/36340264/83040899-71cec380-a05d-11ea-8be3-533322b4b69b.png)


![home(2)](https://user-images.githubusercontent.com/36340264/83040879-6a0f1f00-a05d-11ea-9148-03ff8994a301.png)


- Live classes paragraph in the background and "we bring classroom.....19" are very close to each other,hence unreadable.
- Same is with the second and third slide ,where the former summer wars is clashing with "introducing batches" and  later "online courses" with "learn from home"

2) ### **GOOGLE MAPS NOT WORKING**

![maps](https://user-images.githubusercontent.com/36340264/83041590-45677700-a05e-11ea-9729-1aa3bb932d97.png)

3) ### **ITEMS NOT ALIGNED PROPERLY IN ABOUT US WHY CODING BLOCKS**

![wcb](https://user-images.githubusercontent.com/36340264/83043060-02a69e80-a060-11ea-9e2f-644aab4b8405.png)

 4) ### **REGISTER OAGE NOT WORKING**
[https://codingblocks.com/resources](url)

- Elements not aligned properly and some links are not working.

5)  ### **IDE NOT RUNNING FOR JAVA,KOTLIN,GO languages**
[https://ide.codingblocks.com/](url)

- does not print 'Hello world" for the above languages.

**Comment by krishnasonavane1999** [comment]
Hello, Bhaiya I want to report one issue with the android app. **Issue is:**
 Suppose I played the video and some options are available on the screen like progress-bar, 10sec Pre, 10sec Next, and pause-play. but I observed that all these Icons remain on the screen until we touch again. (I want to say given time is more ). I observed in Hotstar App they have limited time like 2-3 sec and after that, all these icons automatically disappear.

I hope you are getting my point. (Not able to attach screenshots as taking screenshots permission not available)

**Comment by moriarity9211** [comment]
Their is no Rate Limit on Codingblocks Login Through [OTP.](https://account.codingblocks.com/login/otp)
Impact :-- Attacker Might Takeover the user account by bruteforcing  OTP.
![image](https://user-images.githubusercontent.com/46393548/89000332-4dbf8500-d321-11ea-907c-cfd2286606ae.png)
